config/database.yml

```diff
@@ -22,8 +22,12 @@ default: &default
   pool: 5
 
 development:
-  <<: *default
-  database: SubmissionApp_development
+  adapter: postgresql
```

Comment:
I switched this to connect using TCP, this is the way the db should be interacted with, not through the socket. Plus you'd never run a socket in production anyway, so it mirrors deployed environments closer as well.